easybuild/easyconfigs/y/Yambo/Yambo-4.1.3-CrayIntel-2016.11.eb

```diff
@@ -0,0 +1,46 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'MakeCp'
```

Comment:
The application does not provide `make install`, so we need to copy the executables anyway like you do with `MakeCp`, which provides the useful variable `files_to_copy`.